### PR TITLE
Sept updates

### PR DESCRIPTION
**Pull request by froboy**
- Merges /wiki and /docs sections using `aliases` to maintain existing links as best as possible
- Begin to consolidate some duplicate pages in those sections.
- https://yusa.atlassian.net/browse/DS-787 D10 Assistance
- https://yusa.atlassian.net/browse/DS-1035 Alert updates
- https://yusa.atlassian.net/browse/DS-796 Main menu 2nd level nav
- https://yusa.atlassian.net/browse/DS-1041 Grid Content block
- https://yusa.atlassian.net/browse/DS-1034 Facility CT 
- fix wonky markup where [browsers don't like p > a > figure](https://stackoverflow.com/questions/5163294/html5-figure-figcaption-inside-a-paragraph-gives-unpredictable-output#:~:text=The%20figure%20element%20is%20block,it%20to%20allow%20for%20wrap.)

### CODE REVIEW

File: content/en/docs/development/drupal-10-update/_index.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+If you run into an error like this:
+
+> \>  [notice] Update started: y_lb_update_9011
+>
+> \> [error]  Configuration <em class="placeholder">core.entity_view_display.node.lb_event.featured</em> depends on the <em class="placeholder">core.entity_view_mode.node.featured</em> configuration that will not exist after import.
+
+you may be able to resolve it yourself:
+
+1. Search the codebase for the missing config.
```
Comment (krknopp) [inline content/en/docs/development/drupal-10-update/_index.md:99]: can you add an example of how to search?  i.e. `grep -rI lb_event .` from the `docroot` parent folder

File: content/en/docs/development/drupal-10-update/_index.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+2. Add an update hook to your own custom module [like this example](https://github.com/YCloudYUSA/yusaopeny/pull/108/files#diff-2b10287d954ae6b7c36a1af05970dc0f2ee602047b2215d60d9052a15a8819b5R1233-R1240):
```
Comment (krknopp) [inline content/en/docs/development/drupal-10-update/_index.md:100]: which file does this code go in?

File: content/en/docs/development/drupal-10-update/_index.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        /** @var \Drupal\config_import\ConfigImporterService $config_importer */
+        $config_importer = \Drupal::service('config_import.importer');
+        $config_importer->setDirectory($path);
+        $config_importer->importConfigs([
+            'core.entity_view_mode.node.featured',
+        ]);
+    }
+    ```
+
+3. Use [hook_update_dependencies](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Extension%21module.api.php/function/hook_update_dependencies/10.0.x) to ensure this new update runs before the failing one.
```
Comment (krknopp) [inline content/en/docs/development/drupal-10-update/_index.md:114]: which file does this code go in?

**Comment by froboy** [comment]
@krknopp I've replied to all of your comments in https://github.com/YCloudYUSA/yusaopeny_docs/pull/56/commits/60a77fa31cccc9ae87b642a6418fdfaf99ceb99f. Great feedback, thank you. I hope this helps.

File: content/en/docs/development/AddThis.md
Comment (podarok) [inline content/en/docs/development/AddThis.md:1]: should we keep a file for redirect reasons? @froboy

File: content/en/docs/development/drupal-10-update/_index.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+## Pre-checks
+
+1. CKEditor
+    1. If any custom/contrib modules are used, CKE5 should likely be done AFTER your D10 upgrade
+    2. Contrib checks will NOT be found in the next step, be sure to check these manually
+2. Dependency cleanup
+    1. Modules not installed, but in composer.json should be cleaned up to prevent unwanted dependency issues in trying to update.
+3. Admin theme
+    1. If your website uses a [deprecated admin theme](https://www.drupal.org/docs/core-modules-and-themes/deprecated-and-obsolete#s-recommendations-for-deprecated-themes), you should migrate to Claro theme and test the admin experience. If necessary you can keep the deprecated theme as a contrib package but that is not recommended.
```
Comment (podarok) [inline content/en/docs/development/drupal-10-update/_index.md:26]:
```suggestion
    1. If your website uses a [deprecated admin theme](https://www.drupal.org/docs/core-modules-and-themes/deprecated-and-obsolete#s-recommendations-for-deprecated-themes), you should migrate to Claro theme and test the admin experience. If necessary you can keep the deprecated theme as a contrib package but that is not recommended and won't be supported by this distribution.
```

File: content/en/docs/development/AddThis.md
Comment (froboy) [reply content/en/docs/development/AddThis.md:1]: https://github.com/YCloudYUSA/yusaopeny_docs/pull/56/files#diff-00dc53f6a812c4b36d974af813407470dc53121a211168a5400e10778cbf8ba6R6 this redirect covers that url. https://ds-docs.y.org/docs/development/addthis/

File: content/en/docs/development/drupal-10-update/_index.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+   // This also goes in mymodule.install.
+    function mymodule_update_dependencies() {
+      $dependencies['y_lb'][9011] = [
+        'mymodule' => 9000,
+      ];
+    }
+    ```
+
+3. Re-run `drush updb`.
+4. If you run into other missing config, add them to the list to be imported in `update` hook and re-run `updb`.
```
Comment (podarok) [inline content/en/docs/development/drupal-10-update/_index.md:140]:
```suggestion
4. If you run into other missing configs, add them to the list to be imported in `update` hook and re-run `updb`.
5. Consider to backport your customisation which led to the challenge of doing this upgrade in order for it to be covered and tested by  distribution developers.
```

File: content/en/docs/development/drupal-10-update/_index.md (hunk 4 above)
Comment (froboy) [reply content/en/docs/development/drupal-10-update/_index.md:26]:
```suggestion
    1. If your website uses a [deprecated admin theme](https://www.drupal.org/docs/core-modules-and-themes/deprecated-and-obsolete#s-recommendations-for-deprecated-themes), you should migrate to the Claro theme and test the admin experience. If necessary you can keep the deprecated theme as a contrib package but that is not recommended and won't be supported by the distribution.
```

yes and one more tweak i saw.

**Review by podarok — APPROVED** [review]
Great stuff

**Pull request merged**